src/Modules/Finance/index.ts

```diff
@@ -1 +1 @@
-export {FinanceLayout as Finance} from './FinanceLayout';
```

Comment:
Let's not rename exports. Please, leave a TODO.

Reply:
fixed